README.md

```diff
@@ -1,2 +1,22 @@
 # dp-hello-world
 Examples of services generated by the dp-cli tool
+
+##Description
```

Comment:
Think you need a space between ## and D to make it render properly? not showing quite right on github